src/error.rs

```diff
@@ -1,9 +1,17 @@
 use failure;
 use std::result;
+use std::io::{self, ErrorKind};
 
 pub type Error = failure::Error;
 pub type Result<T> = result::Result<T, Error>;
 
+//impl From<io::Error> for Error
```

Comment:
Can we not leave commented code lying about? I know we're going to redo error-handling, but I think its good habit in general to not leave commented out code in the code, and instead put it in an issue